calcom/cal.com
# fix: add org avatar to org booking pages [CAL-2333]

## What does this PR do?

Added the org avatar in booking pages under the org. Also fixed link of user avatars'.

<kbd><img width="1073" alt="image" src="https://github.com/calcom/cal.com/assets/467258/27bce8bf-e1fa-4d31-8b6f-7be833a789ff"></kbd>

Fixes #10689

## Type of change


## How should this be tested?

Go to an event type belonging to a user or team within an org and see the org avatar in the list of members as well as checking the links of those avatars to be correct now.

## Mandatory Tasks
